### ISSUE TITLE
How to print acc while training?

### ISSUE THREAD

**Issue by fire717** [PyTorchLightning/pytorch-lightning#4850]
## 🚀 Feature
Hi, I wanna print acc(or other metrics) while training in the traing bar,(just like Keras).

And for pytorch, I can compute cumulative acc for every training batch and print it,
However, In lightning, maybe I can compute cumulative acc in training_step(), but it seems like I can only print at the val time? How to print it in the training bar?

I am new to pytorch-lightning, so maybe there is something I missed in the docs?

### Motivation
To know the training status at every batch,  not every epoch (like training_epoch_end() ).

possibly related issue: [100](https://github.com/PyTorchLightning/pytorch-lightning/issues/100)

**Comment by fire717** [comment]
Like in Keras:

We set:
`model.compile(loss="categorical_crossentropy",
                        optimizer=opt, 
                        metrics=['acc'])`

Then  can get:
`25/5405 [..............................] - ETA: 1:13:15 - loss: 0.7136 - acc: 0.6556`

And default in lightning:
`Epoch 0:   5%|██▉                          | 63/1353 [00:22<07:50,  2.74it/s, loss=0.71, v_num=20`

Only have loss.

**Comment by justusschock** [comment]
Hey thanks for your question.

In lightning we have the [Metrics Package](https://pytorch-lightning.readthedocs.io/en/stable/metrics.html#metrics) that also describes it's usage and if you log them with `prog_bar=True` they will also appear on your progress bar as described [here](https://pytorch-lightning.readthedocs.io/en/stable/logging.html#automatic-logging).

Hope this answers your question :)

**Comment by fire717** [comment]
Thx, it seems like useful! I am going to have a try!

**Comment by fire717** [comment]
It is a little strange, when I run in version 1.0.8, the bar cannot show acc and loss is nan,
while I change to v1.0.0, the bar can show  loss value  and acc .

**Comment by justusschock** [comment]
cc @teddykoker @ananyahjha93 any idea on this?

**Comment by ananyahjha93** [comment]
I'll have to see what changes have been added in between, nothing that I am aware of should cause this issue.
